.github/workflows/auto-merge.yaml

```diff
@@ -12,8 +12,15 @@ on:
     types:
       - success
 
+permissions:
+  contents: read
+
 jobs:
   automerge:
+    permissions:
```

Comment:
We are using a special token here that is not generated by GitHub during the action run. So no need to grant it special permissions. The default one on the root level should have no permissions at all.

Reply:
Good catch, missed that.